elasticsearch/src/test/resources/zips-mini-nullable.json

```diff
@@ -0,0 +1,6 @@
+{"age" :  21, "name" : "testName1", "score" : 91, "weight" : 65.0}
```

Comment:
This different between zips-mini-nullable and zips-min file should be the NULL value, Others please stay same.

Reply:
OK, I've updated the dataset.